src/lib/profiles/security/WeaveCert.cpp

```diff
@@ -50,6 +51,27 @@ using namespace nl::Weave::Crypto;
 
 extern WEAVE_ERROR DecodeConvertTBSCert(TLVReader& reader, ASN1Writer& writer, WeaveCertificateData& certData);
 
+static void GenerateTBSCertHash(uint8_t* tbsCert, uint32_t tbsCertLen, WeaveCertificateData& cert)
```

Comment:
What does TBS stand for?

Reply:
To Be Signed.  Its a term of art in the X.509 spec.